documents/autoupdate.md

```diff
@@ -39,8 +39,15 @@ To add an `npm` hook to a library, update the `package.json` with configuration
 * `npmName` should map to the name of the library on `npm`
 * `npmFileMap` is a list of files to take from the `npm` tarball and host on cdnjs
 * `basePath` will be ignored when copying over to the CDN
-* `files` is a pattern matcher allowing selection of multiple files
+* `files` is a pattern matcher allowing selection of multiple files, see [node-glob](https://github.com/isaacs/node-glob#glob-primer)
+  * you can use `!(...)` to exclude the files which are not needed. For example, if you want to exclude `*.flow` and `*.scss`, you can use:
 
+  ```js
+  "files": [
+    "**/*/!(*.flow|*.scss)",  //To fetch all files under all the folders except *.flow and *.scss
+    "*.!(flow|scss)"          //To fetch all files in current path except *.flow and *.scss
```

Comment:
Can't these two example use a single style in the glob? `*.!(flow|scss)` or `!(*.flow|*.scss)`?